test/gtest/ucs/test_memtrack.cc

```diff
@@ -115,31 +132,43 @@ UCS_TEST_F(test_memtrack, parse_dump) {
 }
 
 UCS_TEST_F(test_memtrack, malloc_realloc) {
+    unsigned init_count;
+    size_t init_size;
     void* ptr;
 
+    get_total_count_size(init_count, init_size);
```

Comment:
can we convert init_count/init_size to member variables, and do what get_total_count_size() is doint in the test fixture init() function?
will allow removing this code from all tests